python/ml4ir/applications/classification/tests/test_classification_model.py

```diff
@@ -17,22 +17,18 @@ def run_default_pipeline(self, data_format: str):
         # Fix random seed values for repeatability
         self.set_seeds()
 
-        args: Namespace = self.get_overridden_args(data_format)
```

Comment:
I will add it. It was giving me errors while committing because you had not imported Namespace in this module. I think you use an IDE and it doesn't catch these errors because imports from all modules are in same context.